extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.js

```diff
@@ -575,6 +575,20 @@ angular.module('PaperUI.controllers.configuration', [ 'PaperUI.constants' ]).con
     $scope.hasProperties = function(properties) {
         return util.hasProperties(properties);
     }
+    
+    $scope.showDescription=function(channel,channelType){
+        var popup = $mdDialog.alert({
+            title: channel.label?channel.label:channel.channelType?channel.channelType.label:channel.id,
+            textContent: channel.description?channel.description:channel.channelType?channel.channelType.description:'No description available',
```

Comment:
Instead of showing "No description available", might it make more sense to only offer the long press on channels where there is a description available?